package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "growthbeat-core-javascript",
+  "version": "0.1.0",
+  "description": "Growthbeat SDK for JavaScript",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SIROK/growthbeat-core-javascript"
+  },
+  "author": "SIROK, Inc.",
+  "dependencies": {
+    "grunt": "^0.4.5"
+  },
```

Comment:
Add `private` field, If you will publish it in npm, please set `false` as a value.